app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -928,6 +940,7 @@ public String toJSON() {
             jsonObject.put("raw_calculated", raw_calculated);
             jsonObject.put("raw_data", raw_data);
             jsonObject.put("calculated_value_slope", calculated_value_slope);
+            jsonObject.put("calibration_uuid", calibration_uuid);
```

Comment:
By the way, another solution, is to only add the calibration string, when sending a single packet, but not when sending more than one packet.